tests/core/test_validate.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+from rasa.core.validate import Validate
+from rasa.core.domain import Domain
+from rasa_nlu.training_data import TrainingData
+from rasa.core.training.dsl import StoryFileReader
+from rasa.core.training.dsl import StoryStep
+
+domain_file = 'data/test_validate/domain.yml'
+intents_file = 'data/test_validate/intents.md'
+stories_file = 'data/test_validate/stories.md'
+
+validate_test = Validate(domain=domain_file,
```

Comment:
Turn this into a `pytest.fixture` (see other test scripts like `test_interpreter` for examples)